templates/out-of-sync.j2

```diff
@@ -103,7 +103,7 @@ $(document).ready(function () {
                 <span style="display: none"></span>
                 <span class="text-success" title="added">+{{added}}</span>&nbsp;<span class="text-danger" title="removed">-{{removed}}</span>
             </td>
-            <td data-order="{{ d.topol_order }}">&nbsp;</td>
+            <td data-order="{{ -d.depth }}">{{ d.depth }}</td>
```

Comment:
Probably less surprising as
```suggestion
            <td data-order="{{ d.depth }}">{{ d.depth }}</td>
```
otherwise the sort arrows will be in the wrong directions. You can set the default sort order to be by this column if you'd like; there's an example of how to do so in `index.j2`.